src/styles/_catalog.scss

```diff
@@ -1,9 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  justify-items: center;
+  justify-content: space-evenly;
```

Comment:
you can just replace these with justify-content: center

Reply:
i need it for 487px screen and less (so everything for small screen I put at the start of selector and then change some for bigger screens)